.licenserc.yaml

```
@@ -0,0 +1,83 @@
+#
+# Copyright 1999-2019 Seata.io Group.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+header:
+  license:
+    spdx-id: Apache-2.0
+    copyright-owner: Seata.io
+    content: |
+      Copyright 1999-2019 Seata.io Group.
+    
+      Licensed under the Apache License, Version 2.0 (the "License");
+      you may not use this file except in compliance with the License.
+      You may obtain a copy of the License at
+      
+      http://www.apache.org/licenses/LICENSE-2.0
+      
+      Unless required by applicable law or agreed to in writing, software
+      distributed under the License is distributed on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+      limitations under the License.
+
+  paths-ignore:
+    - '.gitignore'
+    - '.gitattributes'
+    - '.travis.yml'
+    - 'codecov.yml'
+    - 'CONTRIBUTING.md'
+    - 'CONTRIBUTING_CN.md'
+    - 'CODE_OF_CONDUCT.md'
+    - 'README.md'
+    - 'LICENSE'
+    - 'NOTICE'
+    - '**/*.md'
+    - '.github/**'
+    #SPI、spring.factories、spring-configuration-metadata.json、additional-spring-configuration-metadata.json
+    - '**/src/test/resources/META-INF/**'
+    - '**/src/main/resources/META-INF/**'
+    - '**/target/**'
+    - '**/*.iml'
+    - 'mvnw'
+    - 'mvnw.cmd'
+    - '*.sh'
+    - 'changes/**'
+    - 'style/**'
+    - 'script/**'
+    - 'seata-plugins/**'
+    - '.mvn/**'
+    - 'sessionStore/**'
+    - 'distribution/LICENSE-BIN'
+    - 'distribution/NOTICE-BIN'
+    - 'test/src/test/resources/**'
+    - 'serializer/seata-serializer-protobuf/src/main/resources/protobuf/io/seata/protocol/transcation/*.proto'
+    - 'saga/seata-saga-statemachine-designer/node_modules/**'
+    - 'node_modules/**'
+    - 'saga/seata-saga-statemachine-designer/**'
+    - '**/src/test/resources/statelang/**'
+    - 'console/src/main/resources/static/console-fe/**'
+    - '**/io/seata/sqlparser/antlr/**/*.g4'
+    - '**/io/seata/sqlparser/antlr/mysql/parser/*'
+    - '**/integration/**/*.proto'
+    - 'console/src/main/resources/static/**'
+    - 'ext/apm-seata-skywalking-plugin/config/agent.config'
+    - 'server/src/main/resources/lua/redislocker/redislock.lua'
+
+
+  comment: on-failure
+  dependency:
+    files:
+      - pom.xml
+      - package.json
```

`dependency` is the top-level section, just like the `header` section

```suggestion
dependency:
  files:
    - pom.xml
    - package.json
```